doc/source/cluster/vms/user-guides/launching-clusters/aws.md

```diff
@@ -1,3 +1,39 @@
+```{note}
```

Comment:
Actually I took a closer look at what's currently in this file. What you have is essentially `## Start Ray with the Ray cluster launcher` so we don't need the extra quick start?